.cloudbees/testing/action.yml

```diff
@@ -71,12 +75,13 @@ runs:
   steps:
     - id: imgbuild
       name: Build and publish Docker image
-      uses: docker://020229604682.dkr.ecr.us-east-1.amazonaws.com/actions/kaniko-action:${{ action.scm.sha }}
+      # uses: docker://020229604682.dkr.ecr.us-east-1.amazonaws.com/actions/kaniko-action:${{ action.scm.sha }}
+      uses: docker://020229604682.dkr.ecr.us-east-1.amazonaws.com/actions/kaniko-action:897eb954bffc9d26fe4b7a4fbe1085335ed4142c
```

Comment:
Revert this line, Keep the existing line

Reply:
I reverted this line in below commit.